src/main/java/frc/robot/subsystems/climb/Climb.java

```diff
@@ -138,20 +138,19 @@ public void periodic() {
     }
   }
 
-  @Override
-  public void initSendable(SendableBuilder builder) {
+@Override
+public void initSendable(SendableBuilder builder) {
     super.initSendable(builder);
-    builder.addStringProperty("Desired Position", (() -> desiredPosition.toString()), null);
+    builder.addStringProperty("Desired Position", () -> desiredPosition.toString(), null);
     builder.addBooleanProperty("At Desired Position", this::atDesiredPosition, null);
-    builder.addDoubleProperty(
-        "Desired Position (Deg)", (() -> climbPositionMap.get(desiredPosition)), null);
-    builder.addDoubleProperty(
-        "Current Left Motor Position (Deg)",
-        () -> climbTalonLeft.getPosition().getValueAsDouble() * 360.0,
-        null);
-    builder.addDoubleProperty(
-        "Current Right Motor Position (Deg)",
-        () -> climbTalonRight.getPosition().getValueAsDouble() * 360.0,
-        null);
-  }
+    builder.addDoubleProperty("Desired Position (Deg)", () -> climbPositionMap.get(desiredPosition), null);
+    builder.addDoubleProperty("Current Left Motor Position (Deg)", () -> climbTalonLeft.getPosition().getValueAsDouble() * 360.0, null);
+    builder.addDoubleProperty("Current Right Motor Position (Deg)", () -> climbTalonRight.getPosition().getValueAsDouble() * 360.0, null);
+    builder.addBooleanProperty("Allow Climb Movement", () -> allowClimbMovement, null);
+    builder.addDoubleProperty("Absolute Encoder Distance (Deg)", () -> climbAbsoluteEncoder.getDistance(), null);
+    builder.addDoubleProperty("Current Slot", () -> currentSlot, null);
+    builder.addDoubleProperty("Trapezoid Setpoint Position (revs)", () -> tSetpoint.position, null);
+    builder.addDoubleProperty("Trapezoid Setpoint Velocity (revs/sec)", () -> tSetpoint.velocity, null);
+}
```

Comment:
can you add these for arm and elevator? and delete the speed one that is currently in elevator?